src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/service/NotificationGeneratorTest.java

```diff
@@ -103,9 +107,18 @@ public class NotificationGeneratorTest {
 
     private String notificationId1 = "notificationId1";
     private String notificationId2 = "notificationId2";
+    private String documentBinaryUrl = "http://host:8080/a/b/c";
+
+    private DocumentTag documentTag = DocumentTag.END_APPEAL;
+
+    Document document = new Document(documentBinaryUrl, documentBinaryUrl, "end-appeal-notice");
+    DocumentWithMetadata documentWithMetadata = new DocumentWithMetadata(document, "desc", null, documentTag);
+    IdValue<DocumentWithMetadata> documentWithMetadataId = new IdValue<>("1", documentWithMetadata);
+    private String refId3 = caseId + "_" + documentTag.name();
+
 
     @BeforeEach
-    public void setup() {
+    public void setup() throws NotificationClientException, IOException {
```

Comment:
Why have you added this throw?

Reply:
For Line 195, getInputStream() call throws IOException